python/ray/serve/schema.py

```diff
@@ -6,13 +6,47 @@
     DeploymentStatusInfo,
     ApplicationStatusInfo,
     ApplicationStatus,
+    DeploymentStatus,
     StatusOverview,
+    DeploymentInfo,
 )
 from ray.serve._private.constants import DEPLOYMENT_NAME_PREFIX_SEPARATOR
 from ray.serve._private.utils import DEFAULT, dict_keys_snake_to_camel_case
 from ray.util.annotations import DeveloperAPI, PublicAPI
 
 
+@DeveloperAPI
```

Comment:
`DeveloperAPI` seems to be misused here. DeveloperAPI indicates a public-facing API that is for advanced users that may change more rapidly than higher-level public APIs (e.g., if we were to allow custom autoscaling plugins).

Reply:
Thanks, removed!